app/lib/zeebe-api/zeebe-api.js

```diff
@@ -143,9 +143,11 @@ class ZeebeAPI {
     const client = await this._getZeebeClient(endpoint);
 
     try {
-      const response = await client.deployProcess({
-        definition: contents,
-        name: this._prepareDeploymentName(name, filePath, diagramType)
+      const response = await client.deployResource({
```

Comment:
I'm actually not sure about this. Is this the code path to deploy only processes, or can you deploy DMN from here?

If so, the code would need to be different. See here: https://github.com/camunda-community-hub/zeebe-client-node-js/blob/multi-tenant-start/src/zb/ZBClient.ts#L728

Reply:
Good find! I'll checkout if this is actually an issue. We do allow deployment of DMN, too :see_no_evil:. But moving to the resource deployment API is an improvement one way or the other :tada: